data/publisher/src/test/java/tech/pegasys/teku/data/publisher/MetricsPublisherTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.data.publisher;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import okhttp3.OkHttpClient;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class MetricsPublisherTest {
+
+  private MockWebServer mockWebServer = new MockWebServer();
+  private OkHttpClient okHttpClient;
+  private MetricsPublisher metricsPublisher;
+
+  @BeforeEach
+  public void beforeEach() throws Exception {
+    mockWebServer.start();
+    okHttpClient = new OkHttpClient();
```

Comment:
I don't think we're using this `OkHttpClient`.

Reply:
it's used as a parameter in the MetricsPublisher contructor.

Reply:
should probably just have it private final and assigned with variable declaration...